jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/HttpConfiguration.java

```diff
@@ -76,7 +76,7 @@ public class HttpConfiguration implements Dumpable
     private CookieCompliance _requestCookieCompliance = CookieCompliance.RFC6265;
     private CookieCompliance _responseCookieCompliance = CookieCompliance.RFC6265;
     private boolean _notifyRemoteAsyncErrors = true;
-    private boolean _relativeRedirectAllowed;
+    private boolean _relativeRedirectAllowed = true;
```

Comment:
(for other reviewers) This changes the default to always use relative redirects (path only).
Prior versions of Jetty would try to always use absolute URIs.

Reply:
I don't mind the default behavior change, but maybe it should be justified with a comment?